pkgs/by-name/lm/lmstudio/darwin.nix

```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     url = "https://releases.lmstudio.ai/darwin/arm64/${version}/LM-Studio-${version}-arm64.dmg";
```

Comment:
- [ ] url needs change including build revision:

https://releases.lmstudio.ai/darwin/arm64/${version}/3/LM-Studio-${version}-arm64.dmg

maybe, we even want to put the build revision into its own variable, as it is present in both URLs now?
https://releases.lmstudio.ai/darwin/arm64/${version}/${build-revision}/LM-Studio-${version}-arm64.dmg

Reply:
Perhaps--but the thing is, they keep changing around the URLs on us (at least for the Linux folks--Mac seems stabler). I'm kinda wondering if we should go the other way and just hardcode the URL every time until they get their act together.